frontend/app/src/components/history/events/HistoryEventNote.vue

```diff
@@ -64,7 +64,7 @@ function isLinkType(t: any): t is keyof ExplorerUrls {
         />
       </template>
       <template v-else-if="note.type === NoteType.WORD && note.word">
-        <span :key="`span-${index}`">{{ note.word }}</span>
+        {{ note.word }}
```

Comment:
This is to reduce the number of DOM elements since these spans don't have an actual reason to exist.